libethereum/Account.cpp

```diff
@@ -48,17 +48,15 @@ AccountMap dev::eth::jsonToAccountMap(std::string const& _json, AccountMaskMap*
 	{
 		Address a(fromHex(account.first));
 		auto o = account.second.get_obj();
-		u256 balance;
+		u256 balance = 0;
 
-		bool haveBalance = true;
+		bool haveBalance = (o.count("wei") || o.count("finney") || o.count("balance"));
 		if (o.count("wei"))
 			balance = u256Safe(o["wei"].get_str());
 		else if (o.count("finney"))
 			balance = u256Safe(o["finney"].get_str()) * finney;
 		else if (o.count("balance"))
```

Comment:
you can do a `if (o.count("wei") ||  o.count("balance")) above, and remove that part.